### DISCUSSION TITLE
Interesting JUCE tutorial series on YouTube

### DISCUSSION THREAD

**Discussion by shooking** [RomanKubiak/ctrlr#117] [General]
https://youtu.be/7n16Yw51xkI

I think it might be of general interest. I started watching it tonight.

**Comment by Ted-juh** [comment]
He started a newer serie of tutorials for Juce 6.0 on his YouTube channel. I think the biggest change is the way Juce handles Namespaces if I'm correct. Because of this, this tutorial might be hard to follow. But for setting up Juce it's a start.
